docs/deploy-development-rp.md

```diff
@@ -74,7 +74,7 @@
      --template-file pkg/deploy/assets/databases-development.json \
      --parameters \
        "databaseAccountName=$DATABASE_ACCOUNT_NAME" \
-       "databaseName=$DATABASE_NAME" \
+       "databaseName=$DATABASE_NAME"
```

Comment:
Is this needed/correct?

Reply:
Yes, the last line should not have a `\`, because it's the end of the command.

Reply:
fixed